client/src/plugins/report-feedback/ClipboardCopySystemInfo.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import Metadata from '../../util/Metadata';
+
+const PLATFORM_MAPPING = {
+  'darwin': 'MacOS',
+  'win32': 'Windows',
+  'linux': 'Linux'
+};
+
+
+export class ClipboardCopySystemInfo {
+
+  constructor(props) {
+    const {
+      getGlobal,
+      activeTab
+    } = props;
+
+    this._backend = getGlobal('backend');
+    this._plugins = getGlobal('plugins');
+    this._config = getGlobal('config');
+    this._activeTab = activeTab;
+  }
+
+  async clipboardCopy(config) {
+    const systemInfo = await this._compileSystemInfoString(config);
+    this._backend.send('clipboard:write-text', { text: systemInfo });
+  }
+
+  async _compileSystemInfoString(config) {
+    const {
+      _appendVersion
+    } = this;
+
+    let systemInfoText = '## Camunda Modeler system information';
+
+    if (config.version) {
+      systemInfoText = appendVersion(systemInfoText, this._getVersion());
+    }
+    if (config.operatingSystem) {
+      systemInfoText = appendOS(systemInfoText, await this._getOS());
+    }
+    if (config.installedPlugins) {
+      systemInfoText = appendPlugins(systemInfoText, this._getPlugins());
+    }
+    if (config.executionPlatform) {
+      systemInfoText = appendExecutionPlatform(systemInfoText, this._getExecPlatform());
+    }
+
+    return systemInfoText;
+  }
+
+  _getVersion() {
+    return Metadata.data.version;
+  }
+
+  async _getOS() {
+    const os = await this._config.get('os.info');
+    return (PLATFORM_MAPPING[os.platform] || os.platform) +
+    '-' + os.release;
```

Comment:
For MacOS, `os.release` actually retrieves the (Darwin) kernel version, not the actual macOS version.

![image](https://user-images.githubusercontent.com/9433996/130463848-fa454c22-4623-4232-9d3c-249cc6b8c123.png)

So instead of macOS `11.5` (Big Sur), it would retrieve `20.6` for me. I don't know whether this is something we have to consider, but there are helpers to map these versions: https://github.com/sindresorhus/macos-release